src/v2/providers/pubsub.ts

```diff
@@ -86,29 +89,30 @@ export class Message<T> {
   }
 }
 
-/** The interface published in a Pub/Sub publish subscription. */
+/** @beta The interface published in a Pub/Sub publish subscription. */
```

Comment:
we should yank beta per our offline discussion

Reply:
Done!